scripts/tool/SetupMultiSynchronisation.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\taoSync\scripts\tool;
+
+use Doctrine\DBAL\Schema\SchemaException;
+use oat\oatbox\extension\InstallAction;
+use oat\taoSync\model\history\byOrganisationId\DataSyncHistoryByOrgIdService;
+use oat\taoSync\model\history\DataSyncHistoryService;
+use oat\taoSync\model\User\HandShakeClientService;
+use oat\taoSync\model\User\ScopedToOrganisationAuthAdapter;
+
+/**
+ * Tool to setup the platform to accept synchronisation from multiple sync manager
+ * Testtaker can connect only if they are part of last synchronisation
+ *
+ * sudo -u www-data php index.php '\oat\taoSync\scripts\tool\SetupMultiSynchronisation'
+ */
+class SetupMultiSynchronisation extends InstallAction
```

Comment:
Please add this in taoActOffline in SetupClientServer since they wanna use this as default.

Reply:
good catch, fixed